packages/cadl-ranch-specs/http/arm/models/resources/main.tsp

```diff
@@ -0,0 +1,38 @@
+import "@azure-tools/cadl-ranch-expect";
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@azure-tools/typespec-client-generator-core";
+import "./toplevel.tsp";
+import "./nested.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+
+@armProviderNamespace
+@service
+@versioned(Versions)
+@doc("Arm Resource Provider management API.")
+namespace Arm.Models.Resources;
```

Comment:
should bve in the Azure namespace and folder

Reply:
updated